app/src/main/java/io/appium/uiautomator2/utils/ScreenshotHelper.java

```diff
@@ -48,7 +48,7 @@
 public class ScreenshotHelper {
     private static final int PNG_MAGIC_LENGTH = 8;
     private static final UiAutomation uia =
```

Comment:
the same should be done to the static member here. UiAutomation instance must be retrieved each time upon request

Reply:
Missed this, updated in the latest commit https://github.com/appium/appium-uiautomator2-server/pull/463/commits/4be80049b1d5c8ebbe2959b4068273eccef725a1